### PR TITLE
add task solution

### PR DESCRIPTION
**Pull request by b-buchkevych**
[DEMO LINK](https://b-buchkevych.github.io/layout_stars/)
[TEST REPORT LINK](https://b-buchkevych.github.io/layout_stars/report/html_report/)

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -0,0 +1,36 @@
+* {
+  margin: 0;
+}
```
Comment (sTorba24) [inline src/style.css:3]: Don't reset margins for *, do it for tags directly, in this case you need it for body only

File: src/style.css [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+.container {
+  height: 96px;
+  width: 96px;
+  background-color: #fff;
+}
```
Comment (sTorba24) [inline src/style.css:9]: you can remove it

File: src/style.css [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+.star {
+  display: flex;
+  justify-content: center;
+  align-items: center;
```
Comment (sTorba24) [inline src/style.css:14]: this too

**Review by sTorba24 — CHANGES_REQUESTED** [review]
GJ:) Lets improve it

**Review by Viktor-Kost — APPROVED** [review]

**Pull request closed without merging**